src/planscape/forsys/views.py

```diff
@@ -14,6 +14,10 @@
                          JsonResponse, QueryDict)
 
 
+# Constants for spatial references.
+kSpatialReferenceForRasters = "+proj=aea +lat_0=23 +lon_0=-96 +lat_1=29.5 +lat_2=45.5 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs"
```

Comment:
Similarly, there's a value for this in src/planscape/planscape/settings.py

Reply:
+1 to Elsie's comment here and below.

Reply:
done : )